### ISSUE TITLE
missing semicolon in generated module yields jshint warning

### ISSUE THREAD

**Issue by rondonjon** [webpack/file-loader#26]
The generated `exports` line has no trailing semicolon, which yields jshint warnings.

```
235:84: Missing semicolon. (code W033)
  >> 	module.exports = __webpack_require__.p + "fa40c1f01e7a257d9ec200925f173201.eot"
```

**Comment by rondonjon** [comment]
Any chance to add this missing character?
I can create a pull request if you prefer :-)

**Comment by sokra** [comment]
Yep. Please create a PR.

**Comment by rondonjon** [comment]
there you go ...

**Comment by SpaceK33z** [comment]
This was fixed in #27.
